### ISSUE TITLE
Add Unit Tests For ChipTactic

### ISSUE THREAD

**Issue by garethellis0** [UBC-Thunderbots/Software#1059]
### Description of the task
See title.

### Acceptance criteria

- [ ] Add unit tests for the `ChipTactic` to improve both line and behavior coverage

### Blocked By
#773

**Comment by garethellis0** [comment]
Duplicate of #815
